src/Player.cpp

```diff
@@ -289,7 +313,7 @@ void Player::DisplayHUD(Enemy *_Enemy) {
     // Displays player's name and health bar. Enemy object is used to print name on the same line as player name for aesthetics.
     // Prints player's name.
     cout << endl;
-    ColourPrint(name, Console::DarkGrey);
+    ColourPrint(name, Console::Grey);
```

Comment:
Why was this changed?

Reply:
Sorry about that, that was left from the testing.